daemon/daemon.go

```diff
@@ -348,13 +348,14 @@ func (daemon *Daemon) restore() error {
 				log.WithError(err).Error("failed to restore container with containerd")
 				return
 			}
+			log.Debugf("alive: %v", alive)
```

Comment:
Is this WIP or intentional addition of debug logs?

Reply:
I found it useful, but could be removed. Anyway, it only shows up with `-D`